Zend/zend_operators.c

```diff
@@ -54,7 +54,38 @@ static _locale_t current_locale = NULL;
 
 #define TYPE_PAIR(t1,t2) (((t1) << 4) | (t2))
 
-static const unsigned char tolower_map[256] = {
+#if __SSE2__
+#define HAVE_BLOCKCONV
```

Comment:
Where does the BLOCKCONV prefix come from?

Reply:
I am introducing it here, for block (case) conversion. Feel free to suggest a better name.